src/dex/maverick-v2/maverick-v2.ts

```diff
@@ -97,41 +104,38 @@ export class MaverickV2 extends SimpleExchange implements IDex<MaverickV2Data> {
     return null;
   }
 
+  async getEventPools(srcToken: Token, destToken: Token) {
+    srcToken = this.dexHelper.config.wrapETH(srcToken);
+    destToken = this.dexHelper.config.wrapETH(destToken);
+
+    return Object.values(this.pools).filter((pool: MaverickV2EventPool) => {
+      return (
+        (pool.tokenA.address.toLowerCase() === srcToken.address.toLowerCase() ||
```

Comment:
Let's put `pool.tokenA.address.toLowerCase()`, `srcToken.address.toLowerCase()`, `pool.tokenB.address.toLowerCase()`  and `destToken.address.toLowerCase()` to constants and use these constants within this condition.

Reply:
fixed